### PR TITLE
TM-3047 Update OWASP dependency

### PR DESCRIPTION
**Pull request by farwahkazmi1-nhs**
## Description
Update OWASP dependency

Ticket: https://nhsd-jira.digital.nhs.uk/browse/TM-3047
Pipeline: https://dev.azure.com/nhsuk/nhsuk.nhsuk-prototype-kit/_build/results?buildId=160462&view=results

### CODE REVIEW

**Review by sikanderakbarali — APPROVED** [review]
Looks good to me

File: src/owasp-dependency-check-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+    <suppress>
+        <notes><![CDATA[ file name: Microsoft Packages && Azure ]]></notes>
```
Comment (mayank1211) [inline src/owasp-dependency-check-suppressions.xml]: Can we move this file to the root directory?

File: src/owasp-dependency-check-suppressions.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <cpe>CVE-2014-9152</cpe>
+        <cve>CVE-2023-44487</cve>
+        <cve>CVE-2021-3918</cve>
+        <cve>CVE-2023-36052</cve>
+        <cve>CVE-2023-36415</cve>
+        <cve>CVE-2020-12762</cve>
+        <cpe>cpe:/a:microsoft:.net_core</cpe>
+        <cpe>cpe:/a:microsoft:asp.net</cpe>
```
Comment (mayank1211) [inline src/owasp-dependency-check-suppressions.xml]: I don't this we need all these Microsoft-related CVEs for this project.

**Review by mayank1211 — APPROVED** [review]
Looks good to me

**Pull request merged**